docs/spec/index.html

```diff
@@ -1,740 +1,934 @@
-        take into account when utilizing this data model.
+        Solutions implementing this specification SHOULD seek conformance with
+        NIST
+        <a
+          href="https://csrc.nist.gov/publications/detail/sp/800-161/rev-1/draft">
+          800-161 Rev. 1 </a> or superceding
+        documents.
       </p>
-    </section>
-
-    <section class="informative">
-      <h2>Ethical and General Design Considerations</h2>
       <p>
-        The W3C Group Note on
-        <a href="https://www.w3.org/TR/design-principles/">
-          Web Platform Design Principles
-        </a> should be utilized as general guidance for contributions to this specification,
-        as well as for developers implementing a solution that is in compliance with
-        or that interacts with solutions implementing this specification.
+        Solutions implementing this specification SHOULD seek conformance with
+        NIST
+        <a href="https://csrc.nist.gov/publications/detail/sp/800-218/draft">
+          800-218 </a> or superceding documents.
       </p>
       <p>
-        In general, when implementing any solution on the web, the developer should
-        have a strong concern that their solution is implemented in an ethical manner
-        that showcases the appropriate and ethical use of technology.  Any such solution
-        should provide a net positive social benefit, and be cognizant of the variety
-        of principles that must be balanced when building and deploying a solution.
-        The W3C TAG document on
-        <a href="https://w3ctag.github.io/ethical-web-principles/">Ethical Web Principles</a>
-        provides a strong enumeration of these principles and considerations for how to
-        best implement a solution with those principles in mind.
+        The
+        <a href="https://nvlpubs.nist.gov/nistpubs/ir/2021/NIST.IR.8397.pdf">
+          Guidelines on Minimum Standards for Developer Verification of Software
+          - NISTIR 8397
+        </a>
+        MUST be followed by developers implementing solutions intended
+        to be conformant with this specification.
+
+        <br /><br />
+        NB: this guidance
+        applies to sections beyond Software Supply Chain issues, and
+        many of the topics covered have discrete sections in this
+        specification or supplemental aids such as the
+        <a
+          href="https://github.com/w3c-ccg/traceability-interop/tree/main/tests/postman">test
+          suite</a>
+        provided in the repository for this specification.
       </p>
     </section>
-  </body>
-</html>
+  </section>
+
+  <section class="informative">
+    <h2>Disaster Recovery</h2>
+    <p>
+      Industry best practices related to disaster recovery should be
+      followed by implementations compliant with this specification,
+      specifically at a minimum guidance from the following whitepapers
+      and specifications SHOULD be used:
+    </p>
+    <ul>
```

Comment:
```suggestion
    <p>
      NB: the list below links out to multiple NIST and other standards,
      many of which are referenced directly in this specification, but
      each of which should be carefully considered and reviewed even if
      not linked to directly from this specification.
    </p>
    <ul>
```

Reply:
@mprorock declining this change or accepting it?

Reply:
declining for now - we have enough formatting changes

Reply:
@OR13 This appears to have already been added to the spec, resolving.